cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/data/ClientConfig.java

```diff
@@ -81,6 +83,14 @@ public class ClientConfig
     public static final String ENABLE_EXPANSION_SHRINK_CHECK_KEY = "enableExpansionShrinkCheck";
     public static final String SIDECAR_PORT = "sidecar_port";
     public static final String QUOTE_IDENTIFIERS = "quote_identifiers";
+    /**
+     * {@code sstable_start_timestamp} and {@code sstable_end_timestamp} are by default inclusive. Both start and end
+     * timestamps are represented in microseconds. To have exclusive bounds, add +1 offset for exclusive start bound
+     * and add -1 for exclusive end bound.
```

Comment:
The comment is confusing to read. Both the range ends are closed/inclusive _always_.

I would just have this.
```suggestion
     * {@code sstable_start_timestamp_micros} and {@code sstable_end_timestamp_micros} define a time range filter
     * for SSTable selection. Both timestamps are represented in microseconds and both bounds are always inclusive
     * (closed range).
```